scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.51.12] - 2021-04-06
+### Changed
+- Do fenzo unassigns in batches outside of the fenzo lock, from @scrosby
```

Comment:
The unassignment is still happening inside the lock, right? This makes it sound like it's happening outside the lock.

Reply:
Reworded.